mllib/src/test/scala/org/apache/spark/mllib/clustering/GaussianMixtureSuite.scala

```diff
@@ -40,10 +41,15 @@ class GaussianMixtureSuite extends FunSuite with MLlibTestSparkContext {
     val seeds = Array(314589, 29032897, 50181, 494821, 4660)
     seeds.foreach { seed =>
       val gmm = new GaussianMixture().setK(1).setSeed(seed).run(data)
+      val sparseGMM = new GaussianMixture().setK(1).setSeed(seed).run(sparseData)
       assert(gmm.weights(0) ~== Ew absTol 1E-5)
       assert(gmm.gaussians(0).mu ~== Emu absTol 1E-5)
       assert(gmm.gaussians(0).sigma ~== Esigma absTol 1E-5)
+      assert(sparseGMM.weights(0) ~== Ew absTol 1E-5)
+      assert(sparseGMM.gaussians(0).mu ~== Emu absTol 1E-5)
+      assert(sparseGMM.gaussians(0).sigma ~== Esigma absTol 1E-5)
```

Comment:
I was thinking that there would be something more to this data, than randomly generated numbers.